### PR TITLE
feat(GuildMember): add dmChannel getter

### PR DESCRIPTION
**Pull request by ImRodry**
**Please describe the changes this PR makes and why it should be merged:**

This PR adds the dmChannel getter to the GuildMember class, similar to the one on the User class. If there's the createDM and deleteDM methods it only makes sense to have this

**Status and versioning classification:**
- Code changes have been tested against the Discord API, or there are no code changes
- I know how to update typings and have done so, or typings don't need updating
- This PR changes the library's interface (methods or parameters added)

### CODE REVIEW

File: packages/discord.js/src/structures/GuildMember.js [hunk 1]
```diff
@@ -210,6 +210,15 @@ class GuildMember extends Base {
     return this.user.id;
   }
 
+  /**
+   * The DM between the client's user and this member
```
Comment (almeidx) [inline packages/discord.js/src/structures/GuildMember.js:214]: Might be worth mentioning that it only returns the channel if it is cached (I know the description is the same as the User class and the private method, but perhaps those should be updated too)

Comment (almeidx) [reply packages/discord.js/src/structures/GuildMember.js:214]: I don't know. One could assume it returns `null` if the channel does not exist at all

Comment (ImRodry) [reply packages/discord.js/src/structures/GuildMember.js:214]: hmm not sure, it's still a getter and all getters rely on cache (or almost all) so I don't think it's necessary. If a maintainer does I can change this and the other descriptions so I'll keep this open until one of them resolves it

**Review by SpaceEEC — APPROVED** [review]

**Review by vladfrangu — APPROVED** [review]
Sure

**Review by iCrawl — APPROVED** [review]

**Pull request merged**